modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfony5ServerCodegen.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
If it's written from scratch, you can remove line 3: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license